pkg/aws/cloud_config_test.go

```diff
@@ -0,0 +1,98 @@
+package aws
+
+import (
+	"github.com/stretchr/testify/assert"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"testing"
+)
+
+func TestCloudConfig(t *testing.T) {
+	cloudConfig := CloudConfig{
+		Region:         "",
+		AccountID:      "",
+		ThrottleConfig: nil,
+	}
+	setupLog := ctrl.Log.WithName("setup")
+
+	type args struct {
+		accountId string
+	}
```

Comment:
nit: i would get rid of this since imo it doesnt add much

```
tests: []struct {
    name string
    wantAccountID string
    accountID string
}{
...
}
...

for _, tt := range tests {
    t.Run(tt.name, func(t *testing.T) {
        cloudConfig.AccountID = tt.accountId
...
        })
    }
```

Reply:
isn't that what runs all the unit tests? Just assuming based on other test classes I found

Reply:
im not sure what you mean by it runs all the unit tests? or do you mean thats how we're doing in the other tests? yeah this is how the other tests do it, but imo its not necessary. all it does is create a nested object inside the test object

```
tests := []struct {
    name string
    args args
    wantAccountID string
}
```

test here is an unamed struct but you could name it like such: 

```
type test struct {
    name string
    args args
    wantAccountID string
}
```

your `args` here is your "arguments" to your tests, but imo its clear that the one argument you need is the account id so it doesnt add that much value for how much boilerplate it requires (like doing `args: args{acccountId:...}` everywhere

that way we can have something like

```
{
    name: "normal case",
    wantAccountID: "123456789012",
    acccountId: "123456789012",
}
{
...
}
```

Reply:
fixed thanks for clarifying I was thinking you were saying to pull out the whole testing looping structure, but I got you. args removed